cas/src/test/java/org/springframework/security/cas/authentication/CasAuthenticationProviderTests.java

```diff
@@ -103,24 +106,70 @@ public void statefulAuthenticationIsSuccessful() throws Exception {
         assertTrue(casResult.getAuthorities().contains(new SimpleGrantedAuthority("ROLE_B")));
         assertEquals(cap.getKey().hashCode(), casResult.getKeyHash());
         assertEquals("details", casResult.getDetails());
+        assertFalse(casResult.isRememberMe());
 
         // Now confirm the CasAuthenticationToken is automatically re-accepted.
         // To ensure TicketValidator not called again, set it to deliver an exception...
-        cap.setTicketValidator(new MockTicketValidator(false));
+        cap.setTicketValidator(new MockTicketValidator(false, null));
 
         Authentication laterResult = cap.authenticate(result);
         assertEquals(result, laterResult);
     }
 
     @Test
+    public void testRememberMeDefaultAttributeName() throws Exception {
+        CasAuthenticationProvider cap = new CasAuthenticationProvider();
+        cap.setAuthenticationUserDetailsService(new MockAuthoritiesPopulator());
+        cap.setKey("qwerty");
+
+        StatelessTicketCache cache = new MockStatelessTicketCache();
+        cap.setStatelessTicketCache(cache);
+        cap.setServiceProperties(makeServiceProperties());
+
+        cap.setTicketValidator(new MockTicketValidator(true, CasAuthenticationProvider.DEFAULT_REMEMBERME_ATTRIBUTE_NAME));
+        cap.afterPropertiesSet();
+
+        UsernamePasswordAuthenticationToken token =
+            new UsernamePasswordAuthenticationToken(CasAuthenticationFilter.CAS_STATEFUL_IDENTIFIER, "ST-124");
+
+        Authentication result = cap.authenticate(token);
+
+        CasAuthenticationToken casResult = (CasAuthenticationToken) result;
+        assertTrue(casResult.isRememberMe());
+    }
+
+    @Test
+    public void testRememberMeNewAttributeName() throws Exception {
```

Comment:
From the documentation of pull requests we shouldn't make smurfnamings: add preffix or suffix for method names with "test", but just say what we test